bzt/jmx/base.py

```diff
@@ -898,16 +898,35 @@ def _get_dur_assertion(timeout):
         return element
 
     @staticmethod
-    def _get_constant_timer(delay):
-        """
+    def get_constant_timer(delay):
+        timer_type = "ConstantTimer"
+        element = etree.Element(timer_type, guiclass="%sGui" % timer_type, testclass=timer_type, testname="Think-Time")
+        element.append(JMX._string_prop("%s.delay" % timer_type, delay))
+        return [element, etree.Element("hashTree")]
 
-        :type delay: int
-        :rtype: lxml.etree.Element
-        """
-        element = etree.Element("ConstantTimer", guiclass="ConstantTimerGui",
-                                testclass="ConstantTimer", testname="Think-Time")
+    @staticmethod
+    def get_uniform_timer(maximum, offset):
```

Comment:
Why "maximum" if it's "range"?

Reply:
Good question for jmeter programmers.. It's called 'maximum' in gui and has 'range' in prop name. As timers have vague props names (compare 'lambda' and 'range' for Poisson) I've decided to use gui names for clear identification them.